README.md

```diff
@@ -30,7 +30,10 @@ cerberus:
         -    openshift-kube-scheduler
         -    openshift-ingress
         -    openshift-sdn
-        -    openshift-ovn-kubernetes
+       #-    openshift-ovn-kubernetes                    # Enable when OVNKubernetes is used as the SDN. Disable openshift-sdn when that is the case
```

Comment:
Can we not just query to determine the SDN in use?

Reply:
yes, I can add that.
so instead of having openshift-sdn or openshift-ovn-kubernetes in the watch_ns list we can have _default-sdn_ and it'll pick the correct sdn

Reply:
uses the string approach here 
https://github.com/openshift-scale/cerberus/blob/cd13ca61bf84c23c7a6ee68c9daa1fc515459a4c/cerberus/kubernetes/client.py#L72